man/leapp.1

```diff
@@ -112,6 +112,16 @@ Skip actions that use Red Hat Subscription Manager. Equivalent to \fB--no-rhsm\f
 For each XFS partition created with \fIftype=0\fR, leapp creates an overlay file in order to proceed. This option sets the size (in MB) of every such file. Defaults to \fB2048\fR.
 .RE
 
+.B LEAPP_SERVICE_HOST
+.RS 4
+Overrides the host of the service that leapp connects to to fetch necessary data files in case they are missing. It is required to specify the used protocol (\fIhttp://\fR or \fIhttps://\fR). Defaults to \fBhttps://cert.cloud.redhat.com\fR.
```

Comment:
```suggestion
Overrides the host of the service to which leapp connects to fetch necessary data files in case they are missing. It is required to specify the used protocol (\fIhttp://\fR or \fIhttps://\fR). Defaults to \fBhttps://cert.cloud.redhat.com\fR.
```